src/postprocessing/Pass.ts

```diff
@@ -0,0 +1,52 @@
+import { OrthographicCamera, PlaneGeometry, Mesh, Material, Renderer } from 'three'
+
+function Pass() {
```

Comment:
I think if we're gonna change the file `Pass` we should change the function `Pass` to a class too?

Reply:
Done I think!